src/views/studio/SelectMembershipView/SelectMembershipView.tsx

```diff
@@ -0,0 +1,135 @@
+import { Multistepper } from '@/components/Dialogs'
+import routes from '@/config/routes'
+import { useActiveUser, useCheckBrowser } from '@/hooks'
+import { StudioCard, StudioHeader } from '@/shared/components'
+import { promisify } from '@/utils/data'
+import { readFromLocalStorage, writeToLocalStorage } from '@/utils/localStorage'
+import React, { useCallback, useEffect, useState } from 'react'
+import { checkPolkadot, getAccountMemberships } from '../SignInView/fakeUtils'
+import { Membership } from '../SignInView/SignInView'
+import { MemberChannelGrid, Wrapper } from './SelectMembershipView.style'
+import { AccountStep, ExtensionStep, TermsStep } from './Steps'
+
+type DialogStep = {
+  step: number
+  isActive?: boolean
+}
+
+const SelectMembershipView = () => {
+  const [memberships, setMemberships] = useState<Membership[]>()
+
+  const browser = useCheckBrowser()
+  const { activeUser } = useActiveUser()
+  const { dialogVisible, openDialog, closeDialog, changeDialogStep, step } = useSteps()
+
+  // temporary
+  const getMemberShips = useCallback(async () => {
+    if (!activeUser?.accountId) {
+      return
+    }
+    const memberships = await getAccountMemberships(activeUser.accountId)
+    setMemberships(memberships)
+  }, [activeUser?.accountId])
+
+  // temporary
+  const checkIfExtensionIsIntalled = useCallback(async () => {
+    const isInstalled = await checkPolkadot()
+    if (isInstalled && step < 1) {
+      changeDialogStep(1)
+    }
+  }, [changeDialogStep, step])
+
+  useEffect(() => {
+    checkIfExtensionIsIntalled()
+    getMemberShips()
+  }, [checkIfExtensionIsIntalled, getMemberShips])
+
+  const steps = [
+    {
+      title: 'Add Polkadot plugin',
+      element: <ExtensionStep browser={browser} />,
+    },
+    {
+      title: 'Create or select a polkadot account',
+      element: <AccountStep onStepChange={changeDialogStep} currentStepIdx={step} />,
+    },
+    {
+      title: 'Accept terms and conditions',
+      element: <TermsStep onStepChange={changeDialogStep} currentStepIdx={step} />,
+    },
+  ]
+
+  const hasMemberships = memberships?.length
+
+  return (
+    <Wrapper>
+      <StudioHeader
```

Comment:
I think we need to make it smaller on mobile devices, doesn't look good on <500px